mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -177,6 +177,17 @@ func.func @test_extract_strided_slice_1(%arg0 : vector<4x8xf32>) -> vector<2x2xf
   return %0 : vector<2x2xf32>
 }
 
+// ALL-LABEL:   func.func @test_extract_strided_slice_scalable(
+// ALL-SAME:    %[[VAL_0:.*]]: vector<4x[8]xf32>) -> vector<2x[8]xf32> {
+func.func @test_extract_strided_slice_scalable(%arg0: vector<4x[8]xf32>) -> vector<2x[8]xf32> {  
```

Comment:
If the original test was `@test_extract_strided_slice_1`, then this should be `@test_extract_strided_slice_1_scalable`